### DISCUSSION TITLE
Windows 10 network can't open

### DISCUSSION THREAD

**Discussion by Diffused7245** [valinet/ExplorerPatcher#444] [General]
![image](https://user-images.githubusercontent.com/76898556/142787974-c775d792-e317-47c7-a3c6-1b07880b9898.png)
Always "Loading",and auto close in few seconds

**Comment by valinet** [comment]
OS build? I have tested on what’s mentioned in the change log, slim chances to work on other builds. Latest Insider Preview doesn’t work, checked on that as well. It’s expected.

**Reply by Diffused7245** [reply]
22504.1010  Insider Preview , Pro Workstation.

**Reply by Diffused7245** [reply]
And laptop battery can't open too (Nothing happens when clicked）

**Reply by valinet** [reply]
Yeah, as I said, doesn’t work there, I have checked it out, they changed it slightly and some adjustments have to be made to the patch.

**Reply by abcpop** [reply]
Are you talking about win 21H2 22000.348 ?

**Reply by abcpop** [reply]
Congratulation Valentin , 22000.318.38.7 is fine with 21H2 22000.348
